### DISCUSSION TITLE
请教下截图问题，还有目前的模拟器推荐

### DISCUSSION THREAD

**Discussion by chiello0** [lisai9093/YYS#18] [Q&A] [answered]
目前雷电用起来特别卡大家用的什么 还想请问下大佬，我mumu多开的时候提示空间不足 是不是运行的时候有很多截图文件，需要清理吗，很多小白问题  抱歉。致敬作者大大 @lisai9093

**Comment by lisai9093** [comment]
目前此脚本应该不支持模拟器多开功能，只能一个模拟器。mumu我之前测试比较卡就换成了雷电。。。

**Reply by chiello0** [reply]
感谢，另外个小白问题   截图识别的话 有截图文件需要定期清理吗呃

**Reply by lisai9093** [reply, marked as answer]
不需要的所有截图都是直接加载到内存里，不会生成文件

**Reply by chiello0** [reply]
感谢大佬，祝生活愉快
